features/zoeken/met-adresseerbaar-object-identificatie/overzicht.feature

```diff
@@ -0,0 +1,76 @@
+# language: nl
+Functionaliteit: zoek personen met de adresseerbaar object identificatie van een adres
+
+  Als consumer
+  wil ik personen vinden door het opgeven van de adresseerbaar object identificatie van het adres waar zij staan ingeschreven
+
+  Regel: personen worden gevonden die staan ingeschreven op het adres waarmee wordt gezocht 
+
+    Scenario: één persoon staat ingeschreven op het adres waarmee wordt gezocht
+      Gegeven adres 'A1'
```

Comment:
@CathyDingemanse in de nieuwe features gebruiken we voor personen echte namen. Willen we dat voor adressen niet ook doen? Zodat het wat vriendelijker en minder technisch overkomt?

Dus in plaats van "Gegeven adres 'A1'" bijvoorbeeld "Gegeven adres 'Voorbeeldstraat 1'" of "Gegeven adres 'Wilhelmina van Pruisenweg 52"

Reply:
Dan moet je in de stap `En de persoon is ingeschreven op adres '[adres aanduiding]'` ook dezelfde aanduiding invullen. Dit wordt naar mijn mening allemaal complexer. Als je een foutmelding krijgt dat een adres met de gegeven aanduiding niet wordt gevonden, moet je gaan uitzoeken waarom het niet wordt gevonden. Staan er twee spaties? Is een karakter als kleine letter opgegeven terwijl het een hoofletter moet zijn? Enz.

Reply:
niet de moeite waarde